### DISCUSSION TITLE
Proposal| Q3 DCDAO Event Budget

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#27] [Polls]
# Purpose

Hi there, as the development of DCDAO, it would like to apply for some funds to support its online campaigns, like Galex, and the education writer program, to make more users know DCDAO.

# Budget

Q3 is expected to spend 180,000 RING for marketing. The expense will be used for online events, community campaigns, and so on.

Here are the expense details as the following:
<img width="289" alt="截屏2023-07-03 18 15 51" src="https://github.com/dcdao/discussion/assets/109792098/fe05d23e-b401-4a5a-8a61-0f0a5ebcffb0">
My receive wallet: 0xf11D5D94d41ffd0339104A74d3E33f808A27c54E（polygon is preferred)

**Comment by SasoLithops** [comment]
Absolutely. I know our main targets are devs, but I think it is time to start with better marketing of Darwinia across ecosystem as well. 

Maybe think of giving some Ring to doing an interview with Kusamarian if/when he available. You reach devs and ecosystem at once. He has huge respect among Dotsama crypto community. And we could use this interview for promotion for a while...

Imo our community tokens should in future be wisely spent for events that bring new community members,not just "reward hunters".

**Comment by hackfisher** [comment]
> Q3 is expected to spend 150,000 RING for marketing

Some consistency issue with the total 180K RING in the table.

**Comment by JuneCribb** [comment]
I believe the budget for marketing doesn't equate to good marketing. 

The type of marketing done with the budget will show results. 

It's a fair budget and I stand for it.

**Reply by SasoLithops** [reply]
Hi welcome!

Probably just me,not sure I understand you well. Are you talking in general or about this proposal?

**Reply by JuneCribb** [reply]
The proposal, I'm for it. 

I saw a comment about inconsistency in budget and I'm saying sometimes, the budget doesn't equate to good marketing, you just have to find the right type of marketing. 

And with the campaign, events and more to add , the budget is fair.

**Reply by Cindy884** [reply]
do you have any good suggestion about the marketing?

**Reply by JuneCribb** [reply]
Regardless of who the targets are, I'll suggest co-hosting space, and hosting offline/ online with other ecosystems and top influencers in the blockchain and introducing DCDAO to them.

Community Calls can be held to help new members under Darwinia's vision. 

We shouldn't be confined to one ecosystem if we want to bring in new and enthusiastic members.

**Reply by SasoLithops** [reply]
Those are good ideas,we hosted many meetings with Dotsama projects(well Megan and Dustin did). Not sure we got many new members from those,but were very educational. It is hard to onboard new members especialy in bear market when price is constantly droping. Also Darwinia is tech. So its product is for developers which should be main marketing target options. I think that going to pokadot events is priority to onboard new projects. Then we can brag about new amazing partnerships😉

I agree with Cindy proposal to educate community about Darwinia.

And we need to find value to holding Ring and Kton tokens beside staking and fees, but that is for another thread...


@Cindy884 Cindy do you know perhaps what the costs are for having interview with top Polkadot influencers? I can ask Kusamarian and KC but they probably charge a lot.
I can also ask Cryptocito. He was in talk with Darwinia once I still run that TG group we talked. He is now moving outside not just covering Cosmos ecosystem and wants to have 100k subs. Last time I talked to him he was however booked 3 months in advance....

**Reply by 2xJDubs** [reply]
So the Kusmarian doesn't charge and I am sure we could get a piece on there with them.  Jay only wants to do in person stuff now but Pnin and a few other of the big guys for Kusmarian will do audio or text stuff.

**Comment by 1022ranji** [comment]
Please add the Event Budget receive address.

**Comment by 2xJDubs** [comment]
After thinking about this more I think the budget is a good start.  We may not see much retention from the Galex campaigns as this was the case with other events of this style but it will generate activity.  I think we need to require joining our TG and Discord and maybe some number of post threshold or something.  Try to get engagement going.  I like the quiz ideas for sure.

**Reply by SasoLithops** [reply]
I like the quiz idea too but the one we did also didn't bring new members. It has to be well thought out. I also like the idea joining TG is one of requierments. Just don't won't hellos and bots spamming chat. Maybe post in TG something you like/dislike about Darwinia to enter. 
Maybe we could also give some rewards from fund to active members(we excluded ). I mean members who did certain ammount of messages in Darwinia TG and Discord and for sticking with Darwinia in bear market. People like CryptoMegassa. I think this would show we care about our community not just new members.

**Comment by Laki389** [comment]
@Cindy884 

> My receive wallet: 0xf11D5D94d41ffd0339104A74d3E33f808A27c54E（polygon is preferred)

I would suggest that the Galex, Writter program, etc., rewards be given at Darwinia chain instead of Polygon.
Why?
That will make more addresses on the Darwinia chain, more activity on-chain. Although Darwinia is one of the Polkadot ecosystem OGs, due to the migration to Darwinia 2 it has only 760 addresses, 5166 transfers and 1612 signed extrinsics..

I'm not naive and I expect that the majority, if not all, of the winners will want to change the prize to USDT, but at least they will learn to add the Darwinia chain to their Metamask wallet, see how easy it is to do and how easy it is to manage funds and that you don't need complicated wallets like polkadot.js, but a commonly known wallet like Metamask.
This has been the practice and in the past, such awards have been given at the Darwinia chain.
